django/utils/functional.py

```diff
@@ -311,6 +308,16 @@ def __deepcopy__(self, memo):
     __contains__ = new_method_proxy(operator.contains)
 
 
+def unpickle_lazyobject(wrapped):
+    """
+    Used to unpickle lazy objects.
+
+    Actually just returns its argument, which will be the wrapped object.
+    """
+    return wrapped
+unpickle_lazyobject.__safe_for_unpickling__ = True
```

Comment:
It's ea8e7fd989095c1444528c8b9808076985d5ea0a, but https://github.com/django/django/commit/4accffd8fdaa59597dc40b47be031e7116c5b2a5 is also quite interesting :-)